### PR TITLE
Updated repository to GitHub and added selection of commit hash for 0.5.0 version

### PR DESCRIPTION
**Pull request by marceliwac**

### CODE REVIEW

File: rgb/Dockerfile [hunk 1]
```diff
@@ -1,6 +1,6 @@
 FROM ubuntu:14.04
 
-MAINTAINER Ben Heasly <benjamin.heasly@gmail.com>
+LABEL authors="Ben Heasly <benjamin.heasly@gmail.com>, Marceli Wac <marceliwac@gmail.com>"
```
Comment (marceliwac) [inline rgb/Dockerfile:3]: `MAINTAINER` is now deprecated

File: spectral/Dockerfile [hunk 2]
```diff
@@ -1,6 +1,6 @@
 FROM ninjaben/mitsuba-rgb
 
-MAINTAINER Ben Heasly <benjamin.heasly@gmail.com>
+LABEL authors="Ben Heasly <benjamin.heasly@gmail.com>, Marceli Wac <marceliwac@gmail.com>"
```
Comment (marceliwac) [inline spectral/Dockerfile:3]: `MAINTAINER` is now deprecated

**Comment by marceliwac** [comment]
@benjamin-heasly Take a look at the changes and let me know if anything needs to be changed.

**Review by benjamin-heasly — APPROVED** [review]
LGTM!

**Pull request merged**